Building release 0.9 fails (undefined reference to `IMG_SavePNG')

Hi there!

I was able to build all previous versions, but for 0.9, I get this on GNU/Linux (aarch64, but it's not a CPU-arch related problem):
```

/usr/bin/ld: ../engine/libengine.a(image_tool.o): in function `fheroes2::Save(fheroes2::Image const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned char)':
image_tool.cpp:(.text+0x48c): undefined reference to `IMG_SavePNG'

```

Of course I can go and comment the problematic line, but... am I missing something here?

Note I build with:

`WITHOUT_AUDIOCD=1 WITH_SDL2=1 WITHOUT_IMAGE=1 WITHOUT_XML=1 make -j4
`

I believe it could be some #ifdef related to WITHOUT_IMAGE that is missing in 0.9?
Thanks!

--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@
 # WITHOUT_MIXER: build without SDL_mixer library
 # WITHOUT_AUDIOCD: disable audio CD support
 # WITHOUT_UNICODE: build without unicode (disable translation and ttf font)
-# WITHOUT_IMAGE: build without SDL_image library (disable cache image, icn2png)
+# FHEROES2_IMAGE_SUPPORT: build with SDL image support
 # WITHOUT_XML: skip build tinyxml, used for load alt. resources
 # WITH_TOOLS: build tools
 # WITHOUT_BUNDLED_LIBS: do not build XML third party library

--- a/VisualStudio/SDL2.props
+++ b/VisualStudio/SDL2.props
@@ -6,7 +6,7 @@
   <ItemDefinitionGroup>
     <ClCompile>
       <AdditionalIncludeDirectories>$(MSBuildThisFileDirectory)..\..\sdl2\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <PreprocessorDefinitions>WITH_TTF;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>WITH_TTF;FHEROES2_IMAGE_SUPPORT;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
       <AdditionalLibraryDirectories>$(MSBuildThisFileDirectory)..\..\sdl2\lib/$(PlatformTarget);%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>

--- a/src/Makefile
+++ b/src/Makefile
@@ -41,7 +41,7 @@ SDL_LIBS := $(SDL_LIBS) -lSDL2_mixer
 endif
 endif
 
-ifndef WITHOUT_IMAGE
+ifdef FHEROES2_IMAGE_SUPPORT
 CFLAGS := $(CFLAGS) -DWITH_IMAGE $(shell libpng-config --cflags) -DWITH_ZLIB
 ifdef FHEROES2_SDL1
 SDL_LIBS := $(SDL_LIBS) -lSDL_image $(shell libpng-config --libs) -lz

--- a/src/engine/image_tool.cpp
+++ b/src/engine/image_tool.cpp
@@ -24,8 +24,11 @@
 #include <SDL_version.h>
 #include <SDL_video.h>
 
+#if defined( FHEROES2_IMAGE_SUPPORT )
 #if SDL_VERSION_ATLEAST( 2, 0, 0 )
+#define FHEROES2_ENABLE_PNG 1
 #include <SDL_image.h>
+#endif
 #endif
 
 namespace
@@ -76,7 +79,7 @@ namespace
             }
         }
 
-#if SDL_VERSION_ATLEAST( 2, 0, 0 )
+#if defined( FHEROES2_ENABLE_PNG )
         int res = 0;
         const std::string pngExtension( ".png" );
         if ( path.size() > pngExtension.size() && path.compare( path.size() - pngExtension.size(), pngExtension.size(), pngExtension ) ) {

